src/ui/public/filter_bar/lib/__tests__/generate_mapping_chain.js

```diff
@@ -1,7 +1,7 @@
 import sinon from 'auto-release-sinon';
 import expect from 'expect.js';
 import ngMock from 'ng_mock';
-import FilterBarLibGenerateMappingChainProvider from 'ui/filter_bar/lib/generate_mapping_chain';
+import { GenerateMappingChainProvider } from 'ui/filter_bar/lib/generate_mapping_chain';
```

Comment:
Hmm, not sure why I did it that way, but does look like I kept the prefixes on all the others, so will switch back to keeping the prefix here!